src/test/kotlin/cacluator/domain/ExpressionFactoryTest.kt

```diff
@@ -0,0 +1,70 @@
+package cacluator.domain
+
+import calculator.domain.Expression
+import calculator.domain.ExpressionFactory
+import calculator.domain.Term
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class ExpressionFactoryTest : BehaviorSpec({
+    given("createExpression에서") {
+        `when`("null이 들어오면") {
+            then("\"0\" expression을 반환한다") {
+                ExpressionFactory.createExpression(null) shouldBe ExpressionFactory.createExpression("0")
+            }
+        }
+
+        `when`("빈문자열이 들어오면") {
+            then("\"0\" expression을 반환한다") {
+                ExpressionFactory.createExpression("") shouldBe ExpressionFactory.createExpression("0")
+            }
+        }
```

Comment:
테스트 꼼꼼하게 잘 작성해주신 것 같아요👍👍

잘 아시고 계시겠지만 Behavior Driven Development(BDD)에서 주로 사용하는 Given/When/Then은 일반적으로 다음과 같이 표현할 수 있는데요,

Given: 주어진 조건, 상황
When: 발생한 사건, 행위
Then: 그로 인한 결과

이 원칙을 따른다면 다음과 같이 표현할 수도 있을 것 같아요.
```suggestion 
    Given("null인 문자열이 있다") {
        val input = null 
        When("주어진 문자열로 Expression을 생성하면") {
            val result = ExpressionFactory.createExpression(input)
            Then("0 expression을 확인할 수 있다") {
                result shouldBe ExpressionFactory.createExpression("0")
            }
        }
    }
```

Reply:
오 given-when-then은 그렇게 쓰는게 맞군요!
저는 given이 테스트하기 위해 주어지는 "대상(객체, 메소드)" 정도로 알고있었는데 주어지는 "상황" 이군요